types/phonetic/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for phonetic 0.1
+// Project: https://github.com/TomFrost/node-phonetic
+// Definitions by: Emmanuel N Kyeyune <https://github.com/emmanuelnk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const phonetic: Phonetic;
+
+export = phonetic;
+
+interface PhoneticOptions {
+    syllables?: number;
+    seed?: string;
+    phoneticSimplicity?: number;
+    compoundSimplicity?: number;
+    capFirst?: boolean;
+}
+
+interface Phonetic {
+  generate: (options: PhoneticOptions) => string;
+}
```

Comment:
```suggestion
export interface PhoneticOptions {
    syllables?: number;
    seed?: string;
    phoneticSimplicity?: number;
    compoundSimplicity?: number;
    capFirst?: boolean;
}

export function generate: (options: PhoneticOptions) => string;
```
it should work with this IMO, there seems to be a just single name function export:
https://github.com/TomFrost/node-phonetic/blob/master/lib/phonetic.js
so while current code works, it just adds additional complexity, what do you think?

Reply:
Good point. I like this change. Tested and working. I'll commit it.